### DISCUSSION TITLE
token_endpoint must be configured on the issuer

### DISCUSSION THREAD

**Discussion by lijiayi0921** [iTwin/itwinjs-core#3426] [Q&A] [answered]
Hi,

I've setup a service to get some information of my imodels. Recently I'm getting this error with my ServiceAuthorizationClient: 
"Authorization error: token_endpoint must be configured on the issuer "

code concerned : 
```
        const authClient = new ServiceAuthorizationClient(iTwinAuthService.AuthConfig);
        // here comes the error :
        const accessToken = await authClient.getAccessToken(); 
```

authConfig with client id, client secret set to my app, scopes are good, authority set to https://ims.bentley.com (setting it or not won't change the outcome).

package concerned : 
@itwin/service-authorization 0.6.2 from npm

How could I fix the problem ?
Thanks
Jiayi

**Comment by aruniverse** [comment, marked as answer]
I just tried updating one of my [test-apps](https://github.com/calebmshafer/itwin-viewer-proxy/pull/12) to use @itwin/service-authorization@0.6.2 and didn't run into any issues. Are you able to repro on 0.6.1? Would you be able to create a small repo where we can reproduce this?

**Reply by lijiayi0921** [reply]
@aruniverse Found the issue.
there was a security alert on the package "minimist" which suggest using 1.2.6.
I forced it with "npm-force-resolutions"
Just tried to empty my node modules and reinstall everything without forced resolution, everything works fine.
Thanks
